LoopKitTests/CarbStoreHKFilterTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CarbStoreHKFilterTests.swift
+//  LoopTests
```

Comment:
These tests belong in a `LoopKitHostedTests` directory and this name should be updated to reflect that.